src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -72,9 +58,28 @@ export class StackDetailsComponent implements OnInit {
     private logger: Logger
   ) { }
 
+  private setStackAnalysisRawData(): void {
```

Comment:
Additionally this function is Private and therefore should be placed below the ngOnInit().